ethereum/contracts/ERC20AtomicSwapper.sol

```diff
@@ -89,61 +81,58 @@ contract ERC20AtomicSwapper {
     /// @param _timelock The number of blocks to wait before the asset can be returned to sender
     /// @param _receiverAddr The ethereum address of the swap counterpart.
     /// @param _bep2Addr The receiver address on Binance Chain
-    /// @param _erc20Amount ERC20 asset to swap out.
+    /// @param _outAmount ERC20 asset to swap out.
     /// @param _bep2Amount BEP2 asset to swap in.
     function initiate(
         bytes32 _secretHashLock,
         uint64  _timestamp,
         uint256 _timelock,
         address _receiverAddr,
         bytes20 _bep2Addr,
-        uint256 _erc20Amount,
+        uint256 _outAmount,
         uint256 _bep2Amount
```

Comment:
Let’s calculate with a constant conversion rate instead of supplying the bep2amount as a parameter